dev-environment/README.md

```diff
@@ -0,0 +1,74 @@
+# Development environment
+
+## Prestart
+* Instal Java 8 JDK
+* Install Docker and Docker Compose
+* Install Intellij Idea
+
+## Run Kafka, Elassandra, Prometheus and Grafana
+### Start containers(required)
+For mac:
+```bash
+docker-compose -f dev-environment/env-mac.yml up -d
+```
+For linux family:
+```bash
+docker-compose -f dev-environment/env.yml up -d
+```
+### Bootstrap Elassandra with keyspaces(required)
+```bash
+docker cp dev-environment/elassandra-bootstrap.cql  elassandra-markets:/elassandra-bootstrap.cql
+docker exec -it elassandra-markets bash
+cqlsh -f elassandra-bootstrap.cql
+```
+Then copy paste all content from elassandra-bootstrap.cql to interactive console and press enter. Write twice exit to 
```

Comment:
remove 24-25 lines, also need fix this in search too.

Reply:
removed